addon/chrome.manifest

```diff
@@ -1 +1 @@
-resource shield-study-rappor .
+content shield-study-rappor .
```

Comment:
Please add a newline at the end of this file.

Reply:
Done